lci-odoo10/odoo/addons/lci_ecole/models/section.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 lorenzo6ioti
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class Section(models.Model):
```

Comment:
as _inherit is "ecole.section" it should be EcoleSection but I can imagine that as it is generated with bob you have this one as you type "section" into the inherit question